src/atwho.js

```diff
@@ -109,7 +109,7 @@ define([
         });
 
         if (options.useRichText) {
-            options.insertTpl = '${name}';
+            options.insertTpl = '${hashtagPath}';
```

Comment:
I think there is some clean up that could be done. https://github.com/dimagi/Vellum/blob/master/src/fuse.js#L67-L78 by using `${name}` here it was inserting `/data/question` instead of  `#form/question` and it was getting converted to hashtag later, which seems silly.

I don't quite remember my thinking here, but it probably had to do with trying to support easy references as a feature flag at first. I think this could be changed to assume easy references instead of not. I'm not sure how deep that will go, so not going to look into it right now